inductiva/simulators/README.md

```diff
@@ -0,0 +1,235 @@
+# Simulators
+
+Inductiva API has available several open-source simulators ready to use. Users that are familiar with the simulators can easily start running simulations with their previously prepared simulation configuration files. In this way, they can take advantage of performant hardware to speed up their simulation and exploration.
+No installation or management of the simulators is required, no need to worry about the hardware, and no need to worry about the software dependencies. Inductiva API takes care of all of that for you.
+
+The simulators currently available are all open-source and have their own dedicated documentation:
+- [SPlisHSPlasH](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH)
+- [DualSPHysics](https://github.com/DualSPHysics/DualSPHysics)
+- [OpenFOAM](https://www.openfoam.com/)
+- [SWASH](https://swash.sourceforge.io/)
+- [XBeach](https://oss.deltares.nl/web/xbeach/)
+- [GROMACS](https://www.gromacs.org/)
+
+Check the documentation of each simulator to learn more on how to configure them. Here, we **highlight** how to use the simulators via Inductiva API and how to scale your simulations with simplicity. There is a general structure that all simulators follow, and then there are some specificities for each simulator.
+
+## Simulators via Inductiva API
+
+To run a simulation you will need to have prepared the configuration files for the simulator you desire to use. This configuration files should be in a folder that will be passed to the simulator and your simulation is ready to start running.
+
+Here, we present an example with DualSPHysics - a Smoothed-Particle Hydrodynamics simulator - and show how it looks in practice. To simplify we provide the configuration files for this example.
+
+### Example
+
+This follows a classical example in CFD with flow around a cylinder.
+
+```python
+import inductiva
+
+inductiva.api_key = "YOUR_API_KEY"
+
+# Download the configuration files into a folder
+input_dir = inductiva.utils.files.download_from_url(
+    "https://storage.googleapis.com/inductiva-api-demo-files/"
+    "dualsph-flow-cylinder.zip"
+)
+
+# Initialize the Simulator
+simulator = inductiva.simulators.DualSPHysics()
+
+# Run simulation with config files in the input directory
+task = simulator.run(input_dir=input_dir)
+```
+
+And that is it! Your simulation is already running on the cloud and you obtain a task object that allows you to manage it. You can check it's status with `task.get_status()`, wait for it to finish with `task.wait()`, and download the results with `task.get_output()`.
```

Comment:
And that's it! Your simulation is now active in the cloud, and you have a `task` object that allows you to manage it.

Reply:
and "check its status"